NUnitConsole.sln.DotSettings

```diff
@@ -7,6 +7,7 @@
 	<s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/ALIGN_MULTIPLE_DECLARATION/@EntryValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/ALIGN_MULTLINE_TYPE_PARAMETER_LIST/@EntryValue">True</s:Boolean>
 	<s:String x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/EMPTY_BLOCK_STYLE/@EntryValue">TOGETHER_SAME_LINE</s:String>
+	<s:String x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/PLACE_ACCESSORHOLDER_ATTRIBUTE_ON_SAME_LINE_EX/@EntryValue">NEVER</s:String>
```

Comment:
I tried to figure out what `PLACE_ACCESSORHOLDER_ATTRIBUTE_ON_SAME_LINE_EX` does, but I only found examples from other github projects (and all has it configured to `NEVER`). So probably ok 😄 .

Reply:
As far as I could figure out, it means you have ReSharper 2017.3 and nothing more. 😄

Reply:
I agree Joseph - see comment below. Sorry!